haiku-apps/batchrename/batchrename-0.2.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="An application and Tracker add-on to rename multiple files at once"
+DESCRIPTION="Renamer is a standalone application, and Tracker add-on for Haiku. \
+You can rename multiple files in multiple directories at once, and also shows \
+a preview. It offers many renaming options, and can also rename files by contents \
+of their attributes, or the result of shell scripts in case you need that."
+HOMEPAGE="https://github.com/axeld/rename"
+COPYRIGHT="2019-2024 pinc Software"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/axeld/rename/archive/b768c8525dbaf15b055ffd0b303b8c79a17d8c9a.tar.gz"
+CHECKSUM_SHA256="d94bd208ca03f46f58ed4daef4a044810630a86df60cf1431912375116a5f229"
+SOURCE_DIR="rename-b768c8525dbaf15b055ffd0b303b8c79a17d8c9a"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Do Tracker add-ons even work for a secondary architecture?

Reply:
Oops, good point. I actually was too lazy to look into why it wouldn't build anymore with GCC2. Please hold off of the recipe until I fixed that, I completely forgot about this.